flink-runtime/src/test/java/org/apache/flink/runtime/scheduler/strategy/PipelinedRegionSchedulingStrategyTest.java

```diff
@@ -67,13 +69,18 @@ private void buildTopology() {
 
 		source = testingSchedulingTopology.addExecutionVertices().withParallelism(PARALLELISM).finish();
 		map = testingSchedulingTopology.addExecutionVertices().withParallelism(PARALLELISM).finish();
+		map2 = testingSchedulingTopology.addExecutionVertices().withParallelism(PARALLELISM).finish();
```

Comment:
I prefer to not change the topology for existing tests and not change unrelated tests.
I think it's better to add a new case `testSchedulingTopologyWithPersistentBlockingEdges()` which initializes a `PipelinedRegionSchedulingStrategy` from a topology which contains `BLOCKING_PERSISTENT` results.

Reply:
You are right. I made the change in 7461070, please have another look.